src/rules/javascript/banned_identifiers.js

```diff
@@ -0,0 +1,17 @@
+import { BANNED_IDENTIFIERS } from 'const';
+
+export default function(context) {
+  return {
+    Identifier: function(node) {
```

Comment:
That makes sense. Possible that it just gets put into the banned identifiers list anyway. :smile: